pages/linux/kwrite.md

```diff
@@ -0,0 +1,20 @@
+# kwrite
+
+> Text editor of the KDE Desktop project.
```

Comment:
What do you think about this?
```suggestion
> Text editor of the KDE Desktop project.
> See also `kate`.
```

Reply:
I haven't used kate. Its not pre installed in kde neon so never used it. Do I add it ?

Reply:
Not sure which version of KDE you're using, but on Manjaro KDE, Kate is preinstalled and Kwrite isn't. But I think we should add it.

Reply:
![image](https://user-images.githubusercontent.com/55500003/135767318-fa9cf285-58a8-4682-a075-5fe913485f47.png)

Reply:
ok i will add that line